packages/cli/src/commands/identity/identifier.ts

```diff
@@ -0,0 +1,84 @@
+import { OdisUtils } from '@celo/contractkit'
+import { AuthSigner, ServiceContext } from '@celo/contractkit/lib/identity/odis/query'
+import { flags as oFlags } from '@oclif/command'
+import { cli } from 'cli-ux'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { printValueMap } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+export default class IdentifierQuery extends BaseCommand {
+  static description =
+    'Queries ODIS for the on-chain identifier and pepper corresponding to a given phone number.'
+
+  static flags = {
+    ...BaseCommand.flags,
+    from: Flags.address({
+      required: true,
+      description: 'The address from which to perform the query',
+    }),
+    phoneNumber: Flags.phoneNumber({
+      required: true,
+      description: 'The phone number for which to query the identifier (w/ country code)',
+    }),
+    env: oFlags.string({
+      required: false,
+      description: 'mainnet (default), alfajores, or alfajores_staging',
+    }),
+  }
+
+  static examples = [
+    'identifier --phoneNumber +14151231234 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631 --env alfajores',
+  ]
+
+  async run() {
+    const { flags } = this.parse(IdentifierQuery)
+    const { phoneNumber, from, env } = flags
+
+    await newCheckBuilder(this, flags.from)
+      .isSignerOrAccount()
+      .runChecks()
+
+    cli.action.start('Querying ODIS for identifier')
+
+    const authSigner: AuthSigner = {
+      authenticationMethod: OdisUtils.Query.AuthenticationMethod.WALLET_KEY,
+      contractKit: this.kit,
+    }
+
+    let serviceContext: ServiceContext
+    switch (env) {
```

Comment:
I considered adding these as constants in OdisUtils. Thoughts?

Reply:
Yeah, I like the idea! You can either define it in this class or in OdisUtils.

Reply:
+1, it would prevent the duplication in mobile/src/network-config.ts too